From 301bbaab5a176ac1250db07925fa73f4a23c1911 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Tue, 30 Mar 2021 15:27:26 +0800
Subject: [PATCH] server: replace ioremap_nocache() with ioremap()

Since the commit 4bdc0d676a64314 (remove ioremap_nocache and devm_ioremap_nocache)
Replace ioremap_nocache() with ioremap().

Upstream-Status: Inappropriate [upstream is still using k5.4 version]

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 services/server/env/linux/osfunc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/server/env/linux/osfunc.c b/services/server/env/linux/osfunc.c
index 59c51dc..c48decb 100644
--- a/services/server/env/linux/osfunc.c
+++ b/services/server/env/linux/osfunc.c
@@ -1052,7 +1052,7 @@ OSMapPhysToLin(IMG_CPU_PHYADDR BasePAddr,
 	switch (ui32MappingFlags)
 	{
 		case PVRSRV_MEMALLOCFLAG_CPU_UNCACHED:
-			pvLinAddr = (void __iomem *)ioremap_nocache(BasePAddr.uiAddr, ui32Bytes);
+			pvLinAddr = (void __iomem *)ioremap(BasePAddr.uiAddr, ui32Bytes);
 			break;
 		case PVRSRV_MEMALLOCFLAG_CPU_WRITE_COMBINE:
 #if defined(CONFIG_X86) || defined(CONFIG_ARM) || defined(CONFIG_ARM64)
-- 
2.17.1

